README.md

```diff
@@ -1 +1,222 @@
+
+<!-- LIVE DEMO -->
+
+## 📽️ Video Walkthrogh <a name="video-walkthroug"></a>
+
+- [Video Walkthrough](https://drive.google.com/file/d/1wDhPkXxslwH0K0BSJIsDzuB5ENz8AlY0/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby >= 3.2.0
+    rails >= 7.0
+    postgres >- 15.3
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/ichsansandy/budget-buddy.git
+```
+
+You need to setup database for these project
+
+```
+  development = budget_buddy_dev
+  test        = budget_buddy_test
+  production  = budget_buddy_prod
+```
+
+or you can use your own database and change the ```config/database.yml```
+
+```yml
+  default: &default
+    adapter: postgresql
+    encoding: unicode
+    pool: 5
+    username: [your_username]
+    password: [your_password]
+    host: localhost
+
+  development:
+    <<: *default
+    database: [your_database_for_development]
+
+  test:
+    <<: *default
+    database: [your_database_for_test]
+
+  production:
+    <<: *default
+    database: [your_database_for_production]
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd budget-buddy
+  bundle install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+to use this project:
+
+```ruby
+   bin/rails server
+```
+
+it will run the the server on ```localhost:3000```
+
+### Test
+
+to run test in these this project:
+
+```ruby
+   rspec
+   rails test or bin/rails test
+```
+
+it will run the all the unit test of these project
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="author"></a>
+
+👤 **Ichsan Sandy**
+
+- GitHub: [@ichsansandy](https://github.com/ichsansandy)
+- Twitter: [@1chsansandy](https://twitter.com/1chsansandy)
+- LinkedIn: [Ichsan Sandy](https://linkedin.com/in/ichsans)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add UI System**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ichsansandy/budget-buddy/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can share this project to your friend
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for this project
+
```

Comment:
- [ ] [REQUIRED] Good job so far! The `README` contains an `"Acknowledgements"` section so that should give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) 🙏🏼

Reply:
Addressed